ROADMAP.md

```diff
@@ -54,14 +54,18 @@ _This section is still a very high-level draft_
 Testground As A Service embodies our long-term vision.
 
 - A single, scalable, installation that one or more organizations can use in all their projects,
+- An easy-to-follow documentation and guidelines for users to jump straight into action,
+  - An example: pages that can hand-hold through the learning curve, thus fostering community-driven tutorials. As well as APIs section that can help navigate through the vast possibilities of testground.
 - The ability to experiment with large-scale networks and simplify the integration testing of "any" network-related code,
+  - An example: experiment with a set of rollup networks that deploy their data to a different DA network. A unified/standardised way of doing network(s)-to-network(s) experiments would eliminate a lof of bike-shedding that future teams will go through. 
 - The ability to track the impact of a change in terms of stability & performance across multiple projects,
   - An example: having the ability to run IPFS benchmarks and simulations with different combinations of libraries. This would help us measure regression and improvements as soon as they occur in upstream dependencies.
 
 Products with similar ideas but specialized in different areas:
 
 - database: [CockroachDB performance tracker](https://cockroachdb.github.io/pebble/?max=local),
 - browser: [Webkit Performance Dashboard](https://perf.webkit.org/v3/)
+- selenium : [Selenium](https://www.selenium.dev/)
```

Comment:
Could you share why selenium is useful as a comparison here?

The idea behind Testground As A Service is a unified platform with a dashboard that lets you track changes to your performance metrics, pinpointed to a single commit (similar to https://perf.webkit.org/v3), and eventually run new experiments.

I know selenium as a programmable browser, like playwright or pupetteer, which is more one tool your test might use in TaaS.

Reply:
> Could you share why selenium is useful as a comparison here?
> 
> I know selenium as a programmable browser, like playwright or pupetteer, which is more one tool your test might use in TaaS.

Selenium/Cypress and others are a common one stop solutions for those devs/qa engineers to develop e2e automated tests on their stack (front+back). Usually, such tests are executed on a job basis(e.g. every RC promotion) and then a team does manual testing if needed later

IMHO, testground can cover those e2e automated tests needs for blockchains. Especially, when we are moving towards a multi/cross chain vision in the industry. In addition, we are heavily relying on manual labour of community members is public testnets to perform checks. Some of the manual labour(especially simple like token transfer between accounts, chains) can be eliminated with testground spinning up a full fledge chain(s) and perform checks without humans' participation during the whole execution process(regression, release, smoke, sanity, etc.).  

Now, we don't have a golden standard in writing e2e big network tests where I can schedule a job with a set of test-plans to execute on every release candidate of XYZ client release.

Once we have the standard, a floodgate of new engineers will chime in to learn about QA not only smart-contract wise testing but protocol testing itself, which I believe is crucial considering that appchains' narrative is gaining momentum